templates/manifest.json

```
@@ -1,106 +1,134 @@
-      "defaultRunOptions": {
-        "build": "latest",
-        "memoryMbytes": 4096,
-        "timeoutSecs": 3600
-      }
-    },
-    {
-      "name": "project_cheerio_crawler_ts",
-      "description": "Project: CheerioCrawler in TypeScript - Standard and up to date template for developing with Cheerio.",
-      "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_cheerio_crawler_ts.zip?raw=true",
-      "defaultRunOptions": {
-        "build": "latest",
-        "memoryMbytes": 512,
-        "timeoutSecs": 3600
-      },
-      "skipOptionalDeps": true
-    },
-    {
-      "name": "project_puppeteer_crawler_ts",
-      "description": "Project: PuppeteerCrawler in TypeScript - Standard and up to date template for developing with Puppeteer.",
-      "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_puppeteer_crawler_ts.zip?raw=true",
-      "defaultRunOptions": {
-        "build": "latest",
-        "memoryMbytes": 4096,
-        "timeoutSecs": 3600
-      }
-    },
-    {
-      "name": "project_playwright_crawler_ts",
-      "description": "Project: PlaywrightCrawler in TypeScript - Standard and up to date template for developing with Playwright.",
-      "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_playwright_crawler_ts.zip?raw=true",
-      "defaultRunOptions": {
-        "build": "latest",
-        "memoryMbytes": 4096,
-        "timeoutSecs": 3600
-      }
-    },
-    {
-      "name": "project_empty",
-      "description": "Project: Empty - Standard and up to date project template with very little boilerplate code.",
-      "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_empty.zip?raw=true",
-      "defaultRunOptions": {
-        "build": "latest",
-        "memoryMbytes": 2048,
-        "timeoutSecs": 3600
-      }
-    }
-  ]
+    "templates": [
+        {
+            "name": "getting_started_node",
+            "label": "Start with Node.js",
+            "description": "Basic template for developing actors on the Apify platform. using JavaScript and Node.js.",
+            "category": "javascript",
+            "technologies": ["nodejs"],
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/getting_started_node.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 256,
+                "timeoutSecs": 360
+            }
+        },
+        {
+            "name": "getting_started_typescript",
+            "label": "Start with TypeScript",
+            "category": "typescript",
+            "technologies": ["nodejs"],
+            "description": "Basic template for developing actors on the Apify platform. using TypeScript and Node.js.",
```

`using TypeScript and Node.js.` - the `Using` should have the capital letters as it is at the beginning of the sentence

```suggestion
            "description": "Basic template for developing actors on the Apify platform. Using TypeScript and Node.js.",
```